mite_selenium/stats.py

```diff
@@ -23,6 +23,7 @@
     ("time_to_last_byte", "seconds"),
     ("tls_time", "seconds"),
     ("total_time", "seconds"),
+    ("js_execution_time", "seconds")
```

Comment:
I commented on this in the other PR as well, but I'll bring it here too: `js_execution_time` isn't part of a network resource request; it's it's own top-level thing and so it needs a separate metric not one nested in here.